src/utils/s3_mods.py

```diff
@@ -0,0 +1,148 @@
+# General functions
```

Comment:
A docstring should be added for the file. E.g., 
"""S3 file mods"""

Reply:
Created